file-browser.lua

```diff
@@ -1482,23 +1482,23 @@ local function setup_extensions_list()
 
     --adding file extensions to the set
     for i=1, #compatible_file_extensions do
-        extensions[compatible_file_extensions[i]] = true
+        extensions[string.lower(compatible_file_extensions[i])] = true
     end
 
     --setting up subtitle extensions
     for i = 1, #subtitle_extensions do
-        extensions[subtitle_extensions[i]] = true
-        sub_extensions[subtitle_extensions[i]] = true
+        extensions[string.lower(subtitle_extensions[i])] = true
+        sub_extensions[string.lower(subtitle_extensions[i])] = true
     end
 
     --adding extra extensions on the whitelist
     for str in string.gmatch(o.extension_whitelist, "([^"..pattern_escape(o.root_seperators).."]+)") do
```

Comment:
```suggestion
    for str in string.gmatch(o.extension_whitelist:lower(), "([^"..pattern_escape(o.root_seperators).."]+)") do
```

Change the whole option string to lowercase instead of doing it for every individual extension

Reply:
Done.